src/main/java/org/learn_java/bot/data/entities/Ban.java

```diff
@@ -0,0 +1,44 @@
+package org.learn_java.bot.data.entities;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Ban {
+  @Id @GeneratedValue private long id;
+  private final long user_id;
+  private final String reason;
+  private final boolean is_premanent;
+  private LocalDateTime unban_date;
+
+  public Ban(long id, String reason, LocalDateTime unban_date) {
+    this.user_id = id;
+    this.reason = reason;
+    this.is_premanent = false;
+    this.unban_date = unban_date;
+  }
+
+  public Ban(long id, String reason) {
+    this.user_id = id;
+    this.reason = reason;
+    this.is_premanent = true;
+  }
+
```

Comment:
Also need a no arg constructor, and getters/setters for all fields

Reply:
Got it